openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx

```diff
@@ -1369,18 +1372,29 @@ export const getEntityLineagePath = (
 
 // Nodes Icons
 export const getEntityNodeIcon = (label: string) => {
-  switch (lowerCase(label)) {
-    case EntityType.TABLE:
+  switch (label) {
+    case capitalize(EntityType.TABLE):
```

Comment:
I didn't understand this changes

Reply:
All the entities before were single words 'Table', and 'Topic' so it was working fine we converted it to lowercase before comparison. In the case of 'SearchIndex' that was converting the whole thing in lowercase. So made this change just to keep transformations based on individuals without making assumptions

Reply:
I'll optimize this